packages/core/src/registries/ElementRegistry.ts

```diff
@@ -17,7 +18,10 @@ const register = new Map<string, ElementRegistrationItem>();
 
 const upgrade = (name: string, definition: ElementConstructor): void => {
   definition.applyThemeStyles(CustomStyleRegistry.get(name));
-  customElements.define(name, definition);
+  const isDefined = customElements.get(name);
+  if (isLocalhost || !isDefined) {
```

Comment:
let's add a comment here: `For localhost, define custom element even when it's been defined already to inform developers about the issue.`

Reply:
Fixed